dev/run_s3_archive.R

```diff
@@ -0,0 +1,39 @@
+logger::log_info("JOB START")
+print(
+  nflreadr::nflverse_sitrep(
+    c("nflversedata", "nflreadr", "piggyback", "minioclient")
+  )
+)
+logger::log_info("Downloading rds files from nflverse-data releases")
+purrr::walk(
+  piggyback::pb_releases("nflverse/nflverse-data")$tag_name,
+  \(x) nflversedata::.nflverse_download_assets(
+    release_tag = x,
+    file_type = ".rds",
+    download_dir = file.path("archive", format(Sys.Date()), "rds")
+  )
+)
+
+logger::log_info("Downloading parquet files from nflverse-data releases")
+purrr::walk(
+  piggyback::pb_releases("nflverse/nflverse-data")$tag_name,
+  \(x) nflversedata::.nflverse_download_assets(
+    release_tag = x,
+    file_type = ".parquet",
+    download_dir = file.path("archive", format(Sys.Date()), "parquet")
+  )
+)
+
+logger::log_info("Installing minioclient")
+minioclient::install_mc()
+minioclient::mc_alias_set(alias = "nfl_cf")
+logger::log_info("Syncing files to S3 via minioclient")
+
+minioclient::mc_cp(
+  "archive",
+  "nfl_cf/nflverse",
+  recursive = TRUE,
+  verbose = TRUE
+)
```

Comment:
The test run showed this uploaded about 1 GiB. The free tier is 10 GB-month / month. Wouldn't we exceed that limit pretty fast unless we remove older archives?

Reply:
Yeah that’s a good point, maybe expire monthly? or move into long term storage and pay a little? 

Also can halve it by only storing RDS for the archive

Reply:
Yeah let's just halve it to RDS for now and see how that goes. It's probably worth paying a little bit of money annually rather than doing a bunch of extra lifting to avoid the cost? We'll see, I guess.